packages/SwingSet/src/kernel/kernel.js

```diff
@@ -1016,7 +1050,7 @@ export default function buildKernel(
         const source = { bundle };
         const vatID = kernelKeeper.allocateUnusedVatID();
         const event = { type: 'create-vat', vatID, source, dynamicOptions };
-        kernelKeeper.addToRunQueue(harden(event));
+        kernelKeeper.addToAcceptanceQueue(harden(event));
```

Comment:
I think I'm unclear on the demarcation criteria that determine which deliveries get initially put on the acceptanceQueue and which get put on the runQueue directly.  My intuition says that `'startVat'` should be one of the latter, but I'm not sure what standard I should be judging on.

Reply:
In the future, the standard I believe should be where the message comes from, and if we know the destination. For kernel initiated events, we could imagine having an "outbound queue" for the kernel with high priority.

Regardless, it's a problem I'm punting on until the next PR, as here I'm taking the stance that everything goes through the acceptance queue first, so that this PR doesn't change any delivery order.